hangman.js

```diff
@@ -1 +1,28 @@
-console.log("welcome to hangman");
\ No newline at end of file
+var guest= prompt('Please enter your Nickname', '');
+if ( (guest=='') || (guest==null) )
+{
+ guest="";
+}
```

Comment:
this if statement probably isnt necessary.  If the prompt is left blank, it will return an empty string regardless.